docs/source/install/deb.rst

```diff
@@ -25,8 +32,11 @@ Setup repositories
 
   .. code-block:: bash
 
+    # Pick OS version from: trusty, jessie, wheezy
+    export DISTRO=trusty
     wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
-    echo "deb https://dl.bintray.com/stackstorm/`lsb_release -cs`_staging stable main" | sudo tee /etc/apt/sources.list.d/st2-stable.list
```

Comment:
Really liked this way, considering all `deb` packages we have in Bintray (`trusty`, `wheezy`, `jessie`).

`export DISTRO` and then unset just after this command looks ugly.
Probably saying that we support `trusty`, `wheezy`, `jessie` would be better.